pkgs/build-support/cc-wrapper/utils.sh

```diff
@@ -24,26 +24,16 @@ badPath() {
 }
 
 expandResponseParams() {
-    local inparams=("$@")
-    local n=0
-    local p
-    params=()
-    while [ $n -lt ${#inparams[*]} ]; do
-        p=${inparams[n]}
-        case $p in
-            @*)
-                local parseResponseFile="@parseResponseFile@"
-                if [ -n "$parseResponseFile" ] ; then
-                    eval "params+=($("$parseResponseFile" "${p:1}"))"
-                else
-                    echo "Response files aren't supported during bootstrapping" >&2
-                    exit 1
-                fi
-                ;;
-            *)
-                params+=("$p")
-                ;;
-        esac
-        n=$((n + 1))
-    done
+    if [ -n "@expand-response-params@" ]; then
+        readarray -d '' params < <("@expand-response-params@" "$@")
```

Comment:
Unlike @ryantrinkle, I'm a bit less exited about forking regardless of whether there is a response file. But I can't think of any problems either, correctness-wise or bootstrapping-wise (it took me a second to realize that second part). I don't care to much, and you two outvote me, so lets keep it.

OTOH, great job with `readarray` and null deliminators. Huge fan of those :).